tools/flaky-test-reporter/github_issue.go

```diff
@@ -427,6 +427,12 @@ func (gi *GithubIssue) processGithubIssueForRepo(rd *RepoData, flakyIssuesMap ma
 	var messages []string
 	var errs []error
 
+	// if flag is not set, do not create an issue at all
+	if !rd.Config.PostIssue {
```

Comment:
It might make more sense to default Post Github Issue, how about renaming `PostIssue` with `SkipGithubIssue`?

Reply:
fixed